### ISSUE TITLE
dorado

### ISSUE THREAD

**Issue by boegel** [vscentrum/vsc-software-stack#149]
* link to support ticket: [#2023032860000411](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=115557) + [#2023042060000906](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=118275) + [#2023051960000096](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=120552) + [#2023062060000813](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=123162)
* website: https://github.com/nanoporetech/dorado
* installation docs: https://github.com/nanoporetech/dorado#installation
* toolchain: `foss/2022a` + CUDA 11.7.0
* easyblock to use: *(see existing easyconfig)*
* required dependencies:
  * *(see existing easyconfig)*
* notes:
  * ...
* effort: *(TBD)*
* other install methods
  * conda: no
  * container image: no
  * pre-built binaries (RHEL8 Linux x86_64): yes (https://github.com/nanoporetech/dorado#installation)
  * easyconfig outside EasyBuild: no

**Comment by smoors** [comment]
dorado is already in the EB repo?

**Comment by boegel** [comment]
@smoors Yes, but only 0.1.1, current version is 0.3.0

**Comment by boegel** [comment]
`libaec` is an extra required dependency for `dorado` 0.3.0, required to fix:

```
Could not find AEC_DLL using the following names: aec
```

Next problem is related to HTSlib:

```
rror: HTSCODECS_VERSION_TEXT undeclared
```

cfr. https://github.com/samtools/htslib/issues/1346

It seems like our `HTSlib` does not include the `htscodecs` component at all...

**Comment by boegel** [comment]
The problem with HTSlib was caused by the vendored version included with dorado 0.3.0, it can be worked around by making sure the HTSlib that is provided as a dependency is used, see:
* https://github.com/easybuilders/easybuild-easyconfigs/pull/18155

**Comment by boegel** [comment]
all done
